charts/victoria-metrics-distributed/values.yaml

```diff
@@ -54,7 +54,11 @@ availabilityZones:
     vmauthQueryPerZone:
       enabled: true
       name: ""
-      spec: {}
+      spec:
+        extraArgs:
+          # -- start from v1.100.0, vmauth can discover backend IPs via periodic DNS,
+          # which can help performing load balancing between vminsert instances
```

Comment:
```suggestion
          # -- starting from v1.100.0, vmauth can discover backend IPs via DNS.
          # This can help performing load balancing between vminsert instances
          # See: https://docs.victoriametrics.com/vmauth/#discovering-backend-ips
```